addons/html_builder/static/tests/save.test.js

```diff
@@ -1,36 +1,38 @@
 import { SnippetsMenu } from "@html_builder/builder/snippets_menu";
-import { setContent } from "@html_editor/../tests/_helpers/selection";
-import { insertText } from "@html_editor/../tests/_helpers/user_actions";
 import { expect, test } from "@odoo/hoot";
 import { animationFrame, click } from "@odoo/hoot-dom";
-import { contains, onRpc, patchWithCleanup } from "@web/../tests/web_test_helpers";
-import { defineWebsiteModels, getEditable, setupWebsiteBuilder } from "./helpers";
+import { contains, patchWithCleanup } from "@web/../tests/web_test_helpers";
+import {
+    defineWebsiteModels,
+    exampleWebsiteContent,
+    getEditable,
+    modifyText,
+    setupSaveAndReloadIframe,
+    setupWebsiteBuilder,
+    wrapElExample,
+} from "./helpers";
 
 defineWebsiteModels();
 
-const websiteContent = '<h1 class="title">Hello</h1>';
-
-const emptyWrap = `<div id="wrap" data-oe-model="ir.ui.view" data-oe-id="539" data-oe-field="arch">${websiteContent}</div>`;
-
 test("basic save", async () => {
     const resultSave = setupSaveAndReloadIframe();
-    const { getEditor } = await setupWebsiteBuilder(getEditable(websiteContent));
+    const { getEditor } = await setupWebsiteBuilder(getEditable(exampleWebsiteContent));
     expect(":iframe #wrap").not.toHaveClass("o_dirty");
     await modifyText(getEditor());
 
     await contains(".o-snippets-top-actions button:contains(Save)").click();
     expect(resultSave.length).toBe(1);
     expect(resultSave[0]).toBe(
-        '<div id="wrap" data-oe-model="ir.ui.view" data-oe-id="539" data-oe-field="arch"><h1 class="title">Hello1</h1></div>'
+        '<div id="wrap" data-oe-model="ir.ui.view" data-oe-id="539" data-oe-field="arch"><h1 class="title">H1ello</h1></div>'
     );
     expect(":iframe #wrap").not.toHaveClass("o_dirty");
     expect(":iframe #wrap").not.toHaveClass("o_editable");
-    expect(":iframe #wrap .title:contains('Hello1')").toHaveCount(1);
+    expect(":iframe #wrap .title:contains('H1ello')").toHaveCount(1);
```

Comment:
Preferred like this otherwise, in case modifying by `Hello1`: `.title:contains('Hello')` is valid